src/components/Pagination/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import IconButton from '../IconButton';
+import { theme } from '../../theme';
+
+const Pagination = props => {
+  // Don't show it if both are disabled
+  if (props.prevDisabled && props.nextDisabled) {
+    return null;
+  }
+
+  const { translations } = props;
+
+  return (
+    <Container className={props.className} position={props.position}>
+      <ArrowIcon
+        data-testid={'pagination-prev'}
+        icon={'IconArrowLeft'}
+        onClick={props.goPrev}
+        css={{ marginLeft: '24px' }}
+        disabled={props.prevDisabled}
+        title={`${translations.previous} ${
+          props.prevDisabled ? `- ${translations.disabled}` : ''
+        }`}
+      />
+
+      <ArrowIcon
+        data-testid={'pagination-next'}
```

Comment:
Is it still a good practice?
I remember KCD saying that it should be the last option to use 🤔

Reply:
Yeah, but there is no other way to select it since it's a icon